iceoryx_hoofs/utility/include/iox/into.hpp

```diff
@@ -94,14 +94,14 @@ struct extract_into_type<lossy<T>>
 /// @param[in] value of type SourceType to convert to DestinationType
 /// @return converted value of SourceType to corresponding value of DestinationType
 template <typename SourceType, typename DestinationType>
-constexpr typename detail::extract_into_type<DestinationType>::type_t from(const SourceType value);
+constexpr typename detail::extract_into_type<DestinationType>::type_t from(const SourceType value) noexcept;
```

Comment:
As written above, we can't guarantee here that with no type used here will not throw. Please derive the exception safety from `SourceType` and `DestinationType` with the help of the `noexcept` operator (`noexcept`-`noexcept` idiom).

Reply:
We are a bit inconsistent right now but to properly fix this all over the code base some work is necessary and I would propose to have this for v4.0 and keep it as is.

Reply:
Makes sense. I added this as an item to the [next developer meetup](https://github.com/eclipse-iceoryx/iceoryx/wiki/2023-02-16-Eclipse-iceoryx-developer-meetup).